index.html

```diff
@@ -42,5 +43,185 @@ <h2 class="Supportingtext Title2">I'm a Software Developer</h2>
+        <a class="project_btn_all project_btn_1 resume_btn" href="">Get My Resume</a>
+      </div>            
+    </div>  
+  </section>
+  
+  <!-- Section Skills -->
+  <section class="back_dark">
+    <span class="line_div_dark"></span>         
+    <div class="skills">            
+      <ul class="ul_skill">
+        <li class="li_skills Title2"><h4><img src="/images/diamond_icon.png" alt="Diamond icon">Languages</h4></li>
+        <li class="li_skills Simpletext">JavaScript</li>
+        <li class="li_skills Simpletext">Ruby</li>
+        <li class="li_skills Simpletext">HTML</li>
+        <li class="li_skills Simpletext">CSS</li>
+      </ul>             
+      <ul class="ul_skill">
+        <li class="li_skills Title2"><h4><img src="/images/rectangle_icon.png" alt="Rectangle icon">Frameworks</h4></li>
+        <li class="li_skills Simpletext">React.js</li>
+        <li class="li_skills Simpletext">Ruby on Rails</li>
+        <li class="li_skills Simpletext">RS Pec</li>
+        <li class="li_skills Simpletext">CapyBara</li>
+      </ul>            
+      <ul class="ul_skill">
+        <li class="li_skills Title2"><h4><img src="/images/circle_icon.png" alt="circle icon">Skills</h4></li>
+        <li Class="li_skills Simpletext">Database Managment</li>
+        <li Class="li_skills Simpletext">Version Control</li>
+        <li Class="li_skills Simpletext">CLI</li>
+        <li Class="li_skills Simpletext">API Design</li>
+        <li Class="li_skills Simpletext">Web Development</li>
+      </ul>
+    </div>               
+  </section>
+
+  <!-- Section: Contact me -->
+  <section class="contact_cnt">
+    <div class="grid_text">
+      <h4 class="Simpletext_dark">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h4>
+    </div>
+    <div class="grid_fields">
+      <input class="input_field" type="text" name="full_name" placeholder="Full name" maxlength="40" required>
+      <input class="input_field" type="email" name="email" placeholder="Email address" required>
+      <textarea class="text_area" name="message" placeholder="Write me something" maxlength="400" required></textarea>        
+      <input class="project_btn_all project_btn_submit" type="submit" value="Get in touch"> 
+    </div>
+  </section>
+
+  <!-- Footer -->
+  <footer>
+    <ul class="Socialmedia iconlist iconlist_footer">
+      <li><a href=""><img class="socialmediaicon" src="images/icon-git.png" alt="Git-Hub Icon"></a></li>
+      <li><a href=""><img class="socialmediaicon" src="images/icon-linkedin.png" alt="Twitter icon"></a></li>
+      <li><a href=""><img class="socialmediaicon" src="images/icon-angelist.png" alt="LinkedIn icon"></a></li>
+      <li><a href=""><img class="socialmediaicon" src="images/icon-twitter.png" alt="Angelist icon"></a></li>
+      <li><a href=""><img class="socialmediaicon" src="images/icon-medium.png" alt="Medium icon"></a></li>
+    </ul>
+  </footer>
```

Comment:
This section is not part of the project requirements. So I am not reviewing it. Next time consider adding only those things in the feature branch which are asked in the project requirements 👍